Backend/app/routes/brand_dashboard.py

```
@@ -0,0 +1,1132 @@
+                "revenue": float(campaign_metrics.get("revenue", 0)),
+                "engagement_rate": campaign_metrics.get("engagement_rate", 0),
+                "roi": 0.0  # Calculate ROI based on budget and revenue
+            }
+            
+            # Calculate ROI
+            if campaign.get("budget") and performance["revenue"]:
+                performance["roi"] = (performance["revenue"] - float(campaign["budget"])) / float(campaign["budget"]) * 100
+            
+            performance_data.append(performance)
+        
+        return performance_data
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)